wiki/Reporting_Bad_Behaviour/Handling_Cheating/en.md

```diff
@@ -1,74 +1,74 @@
-Why am I banned?
-----------------
+# Handling Cheating
 
--   10 silences issued in your userpage.
--   You gave your account to somebody else.
+
+## Why am I banned?
+A user may get banned if they do any of the following:
+-   Continuous misbehaviour regardless of how many infringements they accumulate.
+-   Sharing their account with other users.
 -   Using multiple accounts.
--   Cheating (see below).
+-   Hacking or cheating (see below).
```

Comment:
It just :dawn:ed on my, but add "circumventing a punishment by making a new account" ?

Reply:
Done